web/auth/ldap/user_management.py

```diff
@@ -0,0 +1,144 @@
+import codecs
+import os
+import ldap
+from itsdangerous import TimestampSigner
+from flask_login import login_user, make_secure_token
+
+from fame.core.user import User
+from fame.common.config import fame_config
+from web.auth.ldap.config import ROLE_MAPPING
+from web.views.helpers import user_if_enabled
+
+ldap.set_option(ldap.OPT_X_TLS_NEWCTX, ldap.OPT_X_TLS_DEMAND)
```

Comment:
Maybe there should be a way to provide trusted CA certificates in a file referenced in the configuration.

Reply:
I added a short comment on that to the module documentation.